tests/server.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
```

Comment:
Do we need a custom written server here? Couldn't we just use "grunt connect" instead? Or something like "serve" or "sirv-cli"

Reply:
I also thought about this. I created a custom server because, as you said

> We should try to move away from grunt...

and because I'm not a fan big dependency trees. `sirv` seems nice though, if you prefer it, I'll replace the custom solution.

Reply:
Fixed with commit 315c1ad05617a0608df1f9f165f27565d59f7ff9.